bentoml/artifact/tf_savedmodel_artifact.py

```diff
@@ -115,6 +131,47 @@ def load(self, path):
         return self.pack(loaded_model)
 
 
+class TensorflowKerasSavedModelArtifact(TensorflowSavedModelArtifact):
+    """
+    Abstraction for saving/loading Tensorflow.keras model in tf.saved_model format
+
+    Args:
+        name (string): name of the artifact
+
+    Raises:
+        ImportError: tensorflow 2.0 or above is required for TensorflowSavedModelArtifact
+
+    Example usage:
+
+    >>> import bentoml
+    >>> from bentoml.handlers import JsonHandler
+    >>> from bentoml.artifact import TensorflowKerasSavedModelArtifact
+    >>>
+    >>> @bentoml.env(pip_dependencies=["tensorflow"])
+    >>> @bentoml.artifacts([TensorflowKerasSavedModelArtifact('model')])
+    >>> class TfKerasModelService(bentoml.BentoService):
+    >>>
+    >>>     @bentoml.api(JsonHandler)
+    >>>     def predict(self, json):
+    >>>         input_data = json['input']
+    >>>         prediction = self.artifacts.model.add(input_data)
+    >>>         return prediction.numpy()
+    >>>
+    >>> svc = TfKerasModelService()
+    >>>
+    >>> # Option 1: pack directly with Tensorflow trackable object
```

Comment:
this is no longer Tensorflow trackable object here right? should be `tf.keras.models.Model` or something like it?